pkg/server/server.go

```diff
@@ -57,11 +57,11 @@ func (s *Server) loadAllJobs() {
 		log.Error(err)
 		return
 	}
-
-	log.Debugf("%+v", jobs)
+	log.Debugln("....................loading job.........................")
```

Comment:
those lines are not log lines. remove those unnecessary line. this doesn't revel any information.